index.html

```diff
@@ -33,9 +33,19 @@
 
 <p>An efficient incremental rendering list view for large lists.</p>
 
-<p>Ember.ListView works on mobile phones that support web overflow touch css,
-doesn't work on iOS because iOS doesn't emit scroll events during the momentum
-phase of a scroll nor do the elements know their offset top during that scroll phase.</p>
+<p>
+  <code>Ember.ListView</code> works on major modern browsers and also on major mobile devices (iOS, Android).
+  However, there are known issues with using <code>Ember.ListView</code> on mobile web (if you have a long list
+  and you're touch-scrolling it very fast, you'll see that items in your list start to disappear
+  and after some lag appear again). That happens because mobile browsers do not emit scroll events
```

Comment:
because *some* mobile browsers

it also can happen if the browser is under heavy load, it will just stop emitting some events.